CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ else()
         OUTPUT ${jemalloc_targ_SOURCE_DIR}/Makefile
         DEPENDS ${jemalloc_targ_SOURCE_DIR}/configure)
     add_custom_command(
-        COMMAND make
+        COMMAND make -j$(nproc)
```

Comment:
will this work on MacOS?

// I can see in our CI, that we use something like: `-j $(sysctl -n hw.logicalcpu)`

Reply:
We will see ;-)